diff --git a/printf.c b/printf.c
index 8a700ad..fc111a1 100644
--- a/printf.c
+++ b/printf.c
@@ -820,8 +820,8 @@ static int _vsnprintf(out_fct_type out, char* buffer, const size_t maxlen, const
       }
 
       case 'p' : {
-        width = sizeof(void*) * 2U;
-        flags |= FLAGS_ZEROPAD | FLAGS_UPPERCASE;
+        width = sizeof(void*) * 1U;
+        flags |= /*FLAGS_ZEROPAD |*/ FLAGS_HASH /*| FLAGS_UPPERCASE*/;
 #if defined(PRINTF_SUPPORT_LONG_LONG)
         const bool is_ll = sizeof(uintptr_t) == sizeof(long long);
         if (is_ll) {
